### DISCUSSION TITLE
How to export environment variables for Pixi Project

### DISCUSSION THREAD

**Discussion by TheSkyentist** [prefix-dev/pixi#2034] [Q&A] [answered]
Hello! I'm new to pixi and I'm hoping someone can help me either figure out how to export an environment variable for an entire pixi project. 

For example, I want every task or shell I run with pixi to export a specific environment variable, e.g. "JAX_DISABLE_JIT=1". Is there a clean way to do this, or is there a better approach that I should take when thinking about my project?

**Comment by baszalmstra** [comment, marked as answer]
See the `env` section in the `activation` table here: https://pixi.sh/latest/reference/project_configuration/#the-activation-table

**Reply by TheSkyentist** [reply]
This is great! Thanks for pointing me to the relevant documentation. Cheers!
